src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+  const BASIC_DISCOUNT = 0;
+
+  const price = days * PRICE_PER_DAY;
+  const discount = days >= LONG_TERM ? LONG_TERM_DISCOUNT
+    : days >= SHORT_TERM ? SHORT_TERM_DISCOUNT : BASIC_DISCOUNT;
```

Comment:
Nested ternary operators make your code harder to read. Better use `if` with immediate `return` instead 
```
if (something) {
  return price - discount;
}

return price;
 ```